src/ci/resources/local_application.conf

```diff
@@ -28,5 +28,23 @@ backend {
         job-shell = /bin/sh
       }
     }
+    LocalCacheableRuntimeAttribute {
+      actor-factory = "cromwell.backend.impl.sfs.config.ConfigBackendLifecycleActorFactory"
+      config {
+        include "local_provider_config.inc.conf"
+        runtime-attributes = """
+          Int cacheworthy_attribute
+          Int uncacheworthy_attribute_1
+          Int uncacheworthy_attribute_2
+        """
+        runtime-attributes-for-caching = {
+          cacheworthy_attribute: true
+          cacheworthy_attribute_1: true
```

Comment:
Was it meant to be `uncacheworthy_attribute_1: false`?

Reply:
oops, yes! Well spotted!